test_cli.py

```diff
@@ -109,23 +109,25 @@ def get_info(uuid, key):
     return _run_command([cl, 'info', '-f', key, uuid])
 
 
-def wait_until_running(uuid, timeout_seconds=100):
+def wait_until_state(uuid, expected_state='running', timeout_seconds=100):
     start_time = time.time()
     while True:
-        if time.time() - start_time > 100:
+        if time.time() - start_time > timeout_seconds:
             raise AssertionError('timeout while waiting for %s to run' % uuid)
         state = get_info(uuid, 'state')
-        # Break when running or one of the final states
-        if state in {'running', 'ready', 'failed'}:
-            assert state == 'running', "waiting for 'running' state, but got '%s'" % state
+        # Break when running or one of the final states or expected_state
+        if state in {'running', 'ready', 'failed', expected_state}:
```

Comment:
Add a comment to list of FINAL_STATES.  `killed` should be in here too.

Also, appending `expected_state` to the list of final states + running is weird semantically...because I would have expected to wait until `expected_state` was hit.

Reply:
I used the `FINAL_STATES` defined in `bundle_state.py`. I also separated the expected state into a separate check.